alerts/cloudsql/1p-replication-in-error-state.json

```diff
@@ -0,0 +1,34 @@
+{
+  "displayName": "Replication in Error State",
+  "documentation": {
+    "content": "This Alert fires when any replica has stopped working and has entered an error state. The cause should be investigated. for more information around disaster recovery visit: https://cloud.google.com/sql/docs/mysql/replication/cross-region-replicas",
```

Comment:
"This Alert fires when any replica has stopped working and has entered an error state.  For more information around disaster recovery visit: https://cloud.google.com/sql/docs/mysql/replication/cross-region-replicas"

Reply:
updated